client/actions/editActions.js

```diff
@@ -106,6 +132,18 @@ export const saveArticle = (article) => {
   }
 }
 
+export const toggleSpinner = (isVisible) => {
+  if (isVisible) {
+    $('#edit-sections-spinner').show()
+  } else {
```

Comment:
This is called on 'componentDidMount', so regrettably needs to remain jQuery (until we have SSR)